src/wlserver.cpp

```diff
@@ -69,6 +70,7 @@ struct wlserver_content_override {
 enum wlserver_touch_click_mode g_nDefaultTouchClickMode = WLSERVER_TOUCH_CLICK_LEFT;
 enum wlserver_touch_click_mode g_nTouchClickMode = g_nDefaultTouchClickMode;
 
+static std::set <uint32_t> wlserver_touch_down_ids = {};
```

Comment:
Can we move this into the `wlserver` struct?

Reply:
Yep, Thanks for the review, I will update my PR.

Reply:
The PR is updated. I started getting a black screen after the wlroots updates that merged yesterday.  Compiling fine, and looks correct to me. I  cherry picked these changes to the last stable release `3.11.49` and it is working as expected for me.